python/ml4ir/applications/ranking/tests/data/ranklib_test_data/feature_config.yaml

```diff
@@ -0,0 +1,76 @@
+query_key:
```

Comment:
@mohazahran Recommend renaming the folder to `tests/data/ranklib` instead in line with `csv/` and `tfrecord/`.

Reply:
Also recommend moving this config to `tests/data/configs/...`